syntax = "proto3";

package calculator;

message SqrtRequest {
  int32 number = 1;
}

message SqrtResponse {
  int32 result = 1;
}